docs/guides/opentracing-jaeger.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
I think you could put it under advanced: https://docs.saleor.io/docs/next/advanced/intro/ as it is meant for developers